lib/StripeMethod.js

```diff
@@ -45,7 +45,10 @@ function stripeMethod(spec) {
           urlData[param] = '';
           continue;
         }
-        throw new Error('Stripe: I require argument "' + urlParams[i] + '", but I got: ' + arg);
+
+        var err = new Error('Stripe: I require argument "' + urlParams[i] + '", but I got: ' + arg);
```

Comment:
Nit: It maybe helpful to include the `path` in the error so it's immediately obvious what method is causing the problem.